revscoring/datasources/previous_user_rev_doc.py

```diff
@@ -7,10 +7,10 @@ def process(session, revision_metadata):
     
     if revision_metadata.user_text is not None:
         docs = session.user_contribs.query(user={"EpochFail"},
-                                          properties={'ids'},
-                                          limit=1,
-                                          direction="older",
-                                          start="20140901010101")
+                                           properties={'ids','timestamp'},
+                                           limit=1,
+                                           direction="older",
+                                           start=revision_metadata.timestamp-1)
```

Comment:
Assumes that a user can't save two revisions in the same second.  While this is not idea, it seems to work out in practice.  http://blog.wikimedia.org/2014/12/29/how-we-made-editing-wikipedia-twice-as-fast/